Add refine

PratyushJayachandran opened the pull request:
![refinedotdev](https://github.com/simple-icons/simple-icons/assets/35000807/8617538c-91dc-406b-98ee-05b2865ecd72)

**Issue:** closes #8831

**Similarweb rank:**
The similarweb rank is 3,440. See https://www.similarweb.com/website/refine.dev

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
SVG: Taken from website https://refine.dev/about/
Color: `#0B82F0` Taken from https://github.com/refinedev/refine/blob/next/logo.png

LitoMore commented:
Hi @omeraplak, could you help to review this icon? Does the icon meet your expectations?

omeraplak commented:
> Hi @omeraplak, could you help to review this icon? Does the icon meet your expectations? 

Hey,
Thank you for contacting us! I think @recepkutuk can help us!

PratyushJayachandran commented:
> Hey, Thank you for contacting us! I think @recepkutuk can help us!

Hello @recepkutuk, looking forward to your review before we merge this one.

recepkutuk commented:
Hi @PratyushJayachandran this looks great, but we dropped the blue color on the logo. Here's the link to our latest brand assets: https://www.figma.com/file/y8lhTyumAN0zYI5nKKLoLN/refine.dev-brand-assets?type=design&t=DkDibSQq9GrkqBRE-6

PratyushJayachandran commented:
> Hi @PratyushJayachandran this looks great, but we dropped the blue color on the logo. Here's the link to our latest brand assets: https://www.figma.com/file/y8lhTyumAN0zYI5nKKLoLN/refine.dev-brand-assets?type=design&t=DkDibSQq9GrkqBRE-6

Thank you so much @recepkutuk. I have updated the PR with new source and guidelines and the color changed too
![refinedotdev](https://github.com/simple-icons/simple-icons/assets/35000807/61be9183-bc57-4047-b20a-9601f7e34cb4)

recepkutuk commented:
Thank you @PratyushJayachandran , it's perfect now 👍

adamrusted commented on _data/simple-icons.json:
```diff
@@ -11102,6 +11102,12 @@
+        {
+            "title": "Refine.dev",
+            "hex": "14141F",
+            "source": "https://www.figma.com/file/y8lhTyumAN0zYI5nKKLoLN/refine.dev-brand-assets?type=design&node-id=3588%3A22&mode=design&t=xoSXKRwKow3zJ2MC-1",
+            "guidelines": "https://www.figma.com/file/y8lhTyumAN0zYI5nKKLoLN/refine.dev-brand-assets?type=design&node-id=3588%3A22&mode=design&t=xoSXKRwKow3zJ2MC-1"
```
File no longer exists. We could do with a permalink that the brand redirects to the relevant Figma file.

adamrusted replied:
IMO I think we're best to link to the homepage, as right-clicking on the logo will send you to the Figma file.

adamrusted replied:
@recepkutuk - do you have a permalink which redirects to the latest (like refine.dev/brand) that we can use?
It'd make reviewing in the future a _lot_ easier!

recepkutuk replied:
Sure, here's the link: https://s.refine.dev/refine-brand-assets

adamrusted commented on _data/simple-icons.json:
```diff
@@ -11102,6 +11102,12 @@
+        {
+            "title": "Refine.dev",
+            "hex": "14141F",
+            "source": "https://www.figma.com/file/y8lhTyumAN0zYI5nKKLoLN/refine.dev-brand-assets?type=design&node-id=3588%3A22&mode=design&t=xoSXKRwKow3zJ2MC-1",
+            "guidelines": "https://www.figma.com/file/y8lhTyumAN0zYI5nKKLoLN/refine.dev-brand-assets?type=design&node-id=3588%3A22&mode=design&t=xoSXKRwKow3zJ2MC-1"
```
```suggestion
            "source": "https://s.refine.dev/refine-brand-assets",
            "guidelines": "https://s.refine.dev/refine-brand-assets"
```

PeterShaggyNoble commented on _data/simple-icons.json:
```diff
@@ -11349,6 +11349,12 @@
             "source": "https://www.reebok.com/us"
         },
+        {
+            "title": "Refine.dev",
```
```suggestion
            "title": "Refine",
```

PeterShaggyNoble commented on icons/refinedotdev.svg:
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Refine.dev</title><path d="M13.789.422a4.001 4.001 0 0 0-3.578 0l-8 4A4.0011 4.0011 0 0 0 0 8v8c0 1.515.856 2.9 2.211 3.578l8 4a4.001 4.001 0 0 0 3.578 0l8-4A4.0011 4.0011 0 0 0 24 16V8c0-1.515-.856-2.9-2.211-3.578l-8-4ZM8 8c0-2.209 1.791-4 4-4s4 1.791 4 4v8c0 2.209-1.791 4-4 4s-4-1.791-4-4V8Zm6 0c0 1.105-.895 2-2 2s-2- ...
```
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Refine</title><path d="M13.789.422a4.001 4.001 0 0 0-3.578 0l-8 4A4.0011 4.0011 0 0 0 0 8v8c0 1.515.856 2.9 2.211 3.578l8 4a4.001 4.001 0 0 0 3.578 0l8-4A4.0011 4.0011 0 0 0 24 16V8c0-1.515-.856-2.9-2.211-3.578l-8-4ZM8 8c0-2.209 1.791-4 4-4s4 1.791 4 4v8c0 2.209-1.791 4-4 4s-4-1.791-4-4V8Zm6 0c0 1.105-.895 2-2 2s-2-.895-2-2 .895-2 2-2 2 .895 2 2Z"/></svg>
```

PeterShaggyNoble reviewed (dismissed):
As we have approval from the project maintainers, this one can be merged straight in.
